src/index.html

```diff
@@ -2,6 +2,7 @@
   <head>
     <title>CHAT</title>
     <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
update from master. this line is already there.

Reply:
don't change this line, revert your changes

Reply:
fixed